.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,97 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+  LOCATION: eastus
+  CLUSTER: aro-v4-shared
+
+jobs:
+- job: ⏬ Tear down shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
+
+  container:
+    image: mcr.microsoft.com/azure-cli:latest
+
+  steps:
+  - template: ./templates/template-az-cli-login.yml
+    parameters:
+      workingDirectory: /
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
```

Comment:
you'll need a corresponding secret which works on the RH subscription

Reply:
when purge PR merges, we will have this

Reply:
we already have this.